poll/poll.py

```diff
@@ -140,53 +142,60 @@ def gather_items(self, data, result, noun, field, image=True):
             image_alt = item.get('img_alt', '').strip()
             label = item.get('label', '').strip()
             if not label:
                 if image and not image_link:
                     result['success'] = False
+                    # translators: {noun} is either "Answer" or "Question". {noun_lower} is the lower case of either
+                    # Answer or Question.
                     result['errors'].append(
-                        "{0} has no text or img. Please make sure all {1}s "
-                        "have one or the other, or both.".format(noun, noun.lower()))
+                        self.ugettext("{noun} has no text or img. Please make sure all {noun_lower}s "
+                                      "have one or the other, or both.").format(noun=noun, noun_lower=noun.lower()))
                 elif not image:
                     result['success'] = False
                     # If there's a bug in the code or the user just forgot to relabel a question,
                     # votes could be accidentally lost if we assume the omission was an
                     # intended deletion.
-                    result['errors'].append("{0} was added with no label. "
-                                            "All {1}s must have labels. Please check the form. "
-                                            "Check the form and explicitly delete {1}s "
-                                            "if not needed.".format(noun, noun.lower()))
+                    # translators: {noun} is either "Answer" or "Question". {noun_lower} is the lower case of either
+                    # Answer or Question.
+                    result['errors'].append(self.ugettext("{noun} was added with no label. "
+                                                          "All {noun_lower}s must have labels. Please check the form. "
+                                                          "Check the form and explicitly delete {noun_lower}s "
+                                                          "if not needed.").format(noun=noun, noun_lower=noun.lower()))
             if image_link and not image_alt and self.img_alt_mandatory():
                 result['success'] = False
                 result['errors'].append(
-                    "All images must have an alternative text describing the image in a way that "
-                    "would allow someone to answer the poll if the image did not load.")
+                    self.ugettext("All images must have an alternative text describing the image in a way that "
+                                  "would allow someone to answer the poll if the image did not load."))
             if image:
                 items.append((key, {'label': label, 'img': image_link, 'img_alt': image_alt}))
             else:
                 items.append([key, label])
 
         if not items:
+            # translators: {noun_lower} is the lower case of either Answer or Question.
```

Comment:
@asadiqbal08 Capitalize "translators". The comment could be shortened to "{noun_lower} is either "answer" or "question".